index.bs

```diff
@@ -2419,7 +2419,7 @@ To <dfn>get the browsing context info</dfn> given |context|,
 1. Let |context info| be a [=/map=] matching the
    <code>browsingContext.Info</code> production with the <code>context</code>
    field set to |context id|, the <code>parent</code> field set to |parent id|
-   if |is root| is <code>true</code>, or unset otherwise, the <code>url</code>
+   if |is root| is <code>false</code>, or unset otherwise, the <code>url</code>
```

Comment:
why do we need |is root|? it |is root| being true the same as |parent id| being null?

Reply:
ah, that clears this up then, thanks!